### ISSUE TITLE
[Docs]: "React Query (Remote) Example" fails to fetch

### ISSUE THREAD

**Issue by Pirulax** [KevinVandy/mantine-react-table#382]
### mantine-react-table version
v2

### react & react-dom versions
Irrelevant

### Describe the bug and the steps to reproduce it

First of all, not sure if this belongs here, but I'm unsure where to report it.

In the docs [the example](https://v2.mantine-react-table.com/docs/examples/react-query#react-query-(remote)-example) fails to fetch the data.
In the console the following can be seen:
```txt
Access to fetch at 'https://www.mantine-react-table.com/api/data?start=0&size=10&filters=%5B%5D&filterModes=%7B%22firstName%22%3A%22contains%22%2C%22lastName%22%3A%22contains%22%2C%22address%22%3A%22contains%22%2C%22state%22%3A%22contains%22%2C%22phoneNumber%22%3A%22contains%22%7D&globalFilter=&sorting=%5B%5D' from origin 'https://v2.mantine-react-table.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```
The URL blocked is coincidentally the URL that is to be fetched for the table.

Also, not wanting to open a separate issue, but the library uses Mantine V7, so the description should say `v7` not `v5`, no?
> A fully featured Mantine V5 implementation of TanStack React Table V8, forked from Material React Table

### Minimal, Reproducible Example - (Optional, but Recommended)

See the [example in the docs](https://v2.mantine-react-table.com/docs/examples/react-query#react-query-(remote)-example)

### Do you intend to try to help solve this bug with your own PR?

No, because I do not know how

### Terms

- [X] I understand that if my bug cannot be reliably reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.
